asf_public_discourse_home_decarbonisation/pipeline/faqs_identification/extract_questions.py

```diff
@@ -0,0 +1,445 @@
+"""
+This script is designed to extract questions and specific patterns (that could've been written as questions) from a given forum text dataset.
+Key Functionalities:
+    - Extract Questions: Extract Questions: Identifies and extracts questions from sentences based on criteria such finishing with a question mark or starting with a question token (e.g. what, why). Additionally, we also have filtering criteria such as length of sentence and number of tokens.
```

Comment:
Duplicated expression
```suggestion
    - Extract Questions: Identifies and extracts questions from sentences based on criteria such finishing with a question mark or starting with a question token (e.g. what, why). Additionally, we also have filtering criteria such as length of sentence and number of tokens.
```